NeuroID/NIDCallStatusObserver.swift

```diff
@@ -19,22 +19,45 @@ class NIDCallStatusObserver: NSObject, CXCallObserverDelegate {
     }
     
     func callObserver(_ callObserver: CXCallObserver, callChanged call: CXCall) {
-        if call.hasEnded {
-            UtilFunctions.captureCallStatusEvent(eventType: NIDEventName.callInProgress, status: CallInProgress.INACTIVE.rawValue)
-            NIDLog.d("Call has ended")
-        } else if call.isOutgoing {
-            UtilFunctions.captureCallStatusEvent(eventType: NIDEventName.callInProgress, status: CallInProgress.ACTIVE.rawValue)
-            NIDLog.d("Ongoing call observed")
-        } else if call.hasConnected {
-            // Event not captured
-            NIDLog.d("Call connected")
-        } else if call.isOnHold {
-            // Event not captured
-            NIDLog.d("Call on hold")
-        } else {
-            UtilFunctions.captureCallStatusEvent(eventType: NIDEventName.callInProgress, status: CallInProgress.ACTIVE.rawValue)
-            NIDLog.d("Incoming Call observed")
-        }
+        // Outgoing call answered
+        if(call.isOutgoing && call.hasConnected && !call.hasEnded){
+            UtilFunctions.captureCallStatusEvent(eventType: NIDEventName.callInProgress, status: CallInProgress.ACTIVE.rawValue, attrs: Attrs(n: "callMeta", v: CallInProgressMetaData.OUTGOING_ANSWERED.rawValue) )
+            NIDLog.d("Outgoing call (answered) observed /could be voice mail")}
+        
```

Comment:
attrs or attr (I can't remember which) is an array, consider having two properties that we could just stick the "outgoing/incoming" in one and "ringing/answered/ended/onHold" for a second property and reduce the logic needed here?